### DISCUSSION TITLE
Starlight Feature Requests! 🌟

### DISCUSSION THREAD

**Discussion by delucis** [withastro/starlight#968] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to collect ideas for new Starlight features. We hope that you:

* [Suggest features](https://github.com/withastro/starlight/discussions/new?category=feature-requests) you’d like to see added to Starlight 💡 

* Add feedback and context to proposed features 💬 

* **Upvote your favourite proposed features** to help us prioritise what people need 🗳️ 

* Welcome others and are open-minded. Remember that this is a community we build together 💪

**Comment by mrzachnugent** [comment]
Hi, I have a request (maybe it is already possible and  I just missed it 🤷‍♂️)

In the astro config, it would be nice to have I18n for the `title`.

### Something like

```js
starlight({ title: { en: "English" , fr: "Français" } })
```

**Reply by delucis** [reply]
Yes, this would be good to add — we had an early PR for this, but then decided we didn’t like the API and never got round to building something better (the plan was for it to look a bit like your code).

Could you share this as a [new post](https://github.com/withastro/starlight/discussions/new?category=feature-requests) in Discussions? That helps us keep track of things and lets people upvote the ideas.

**Comment by torn4dom4n** [comment]
I have some feature requests. The ideas are from [Nextra](https://nextra.site/):
- Support [Mermaid](https://mermaid.js.org/) diagrams
- Support LaTeX

**Reply by delucis** [reply]
Thanks for the ideas! These are both things we likely wouldn’t add by default, but might both already possible with [Markdown plugins](https://docs.astro.build/en/guides/markdown-content/#markdown-plugins).

By LaTeX, what exactly would you want? For example, here is a demo of support for LaTeX-style maths formulas: https://stackblitz.com/edit/github-xwtnqf?file=src%2Fcontent%2Fdocs%2Findex.md

For Mermaid diagrams, I believe this should be possible using [`rehype-mermaid`](https://github.com/remcohaszing/rehype-mermaid) although I haven’t personally tried setting that up.

**Comment by delucis** [comment]
Thanks for the people posting here! Please post a new discussion for each feature request — this helps us keep track of things and lets people upvote the ideas.
